### ISSUE TITLE
Unexpected warning with the `itemprop` attribute when using the `invalid-attr` rule

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#800]
- Markuplint version: `3.6.0`
- Parser lang: Pug
- Node.js version: `18.x`
- OS: macOS and Windows

## Code Example

```pug
.c-nav-breadcrumb
	ol(itemscope itemtype="https://schema.org/BreadcrumbList")
		li(itemscope itemprop="itemListElement" itemtype="https://schema.org/ListItem" data-breadcrumb="home")
			a(href="/" itemscope itemprop="item" itemtype="https://schema.org/WebPage" itemid="/")
				span(itemprop="name") Home
			meta(itemprop="position" content="1")
			span.c-nav-breadcrumb__separetor
		li(itemscope itemprop="itemListElement" itemtype="https://schema.org/ListItem")
			a(href="/first/" itemscope itemprop="item" itemtype="https://schema.org/WebPage" itemid="/first/")
				span(itemprop="name") Parent
			meta(itemprop="position" content="2")
			span.c-nav-breadcrumb__separetor
		li(itemscope itemprop="itemListElement" itemtype="https://schema.org/ListItem" data-breadcrumb="current")
			a(href="/first/about" itemscope itemprop="item" itemtype="https://schema.org/WebPage" itemid="/first/about")
				span(itemprop="name") Current
			meta(itemprop="position" content="3")
```

## Steps To Reproduce

1. Set true to the `invalid-attr` rule.

## The current behavior

Markuplint said:
> The "itemprop" attribute expects space-separated list. the the content of the list part includes unexpected characters. It expects either "absolute URL", "the property name format" (https://html.spec.whatwg.org/multipage/microdata.html#names:-the-itemprop-attribute) (invalid-attr)

And more.

## The expected behavior

No warning.
